File: src/main/java/org/lvtinger/paladin/message/repo/MessageRepository.java
package org.lvtinger.paladin.message.repo;

import org.lvtinger.paladin.message.api.entity.Message;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface MessageRepository extends JpaRepository<Message, Long> {
    List<Message> getMessageByOwner(Long owner);
    List<Message> getMessageByUniqueId(String uniqueId);
}

File: src/main/kotlin/org/risesun/paladin/provider/impl/repository/MessageRepository.kt
package org.risesun.paladin.provider.impl.repository

import org.risesun.paladin.provider.api.entity.Message
import org.springframework.data.jpa.repository.JpaRepository

interface MessageRepository : JpaRepository<Message, String> {
    fun findBySerialId(serializeId: String): MutableList<Message>
    fun findByMember(member: Long): MutableList<Message>
}